src/util/Parameters.h

```diff
@@ -170,6 +177,9 @@ using SizeT = Parameter<size_t, szt, toString, Name>;
 template <ParameterName Name>
 using String = Parameter<std::string, std::identity, std::identity, Name>;
 
+template <ParameterName Name>
+using Bool = Parameter<bool, bl, toString, Name>;
```

Comment:
Also implement a custom to string converter for `bool` s.t. it is printed as "true" or "false" and not `1` and `0` for better usability.